backend/apis/reports/zipcode-report.api.js

```diff
@@ -0,0 +1,46 @@
+const { app, invalidRequest, pool, databaseError } = require("../../server");
+const { checkValid, nullableValidDate } = require("../utils/validation-utils");
+const mysql = require("mysql");
+const _ = require("lodash");
+const ClientCapability = require("twilio/lib/jwt/ClientCapability");
+
+app.get(`/api/reports/zipcode-report`, (req, res) => {
```

Comment:
A couple conventions we use in this project:

1. nouns in API endpoints are plural
2. no need to put in `-report` since already have `/reports` earlier in the URL

```suggestion
app.get(`/api/reports/client-zipcodes`, (req, res) => {
```

Reply:
Yes, good catch. The singular vs plural consistency tripped me up.